src/misclibs.F

```diff
@@ -3013,12 +3013,14 @@ subroutine calccfl(nstat,rstat,dt,acfl,uh,vh,mh,ua,va,wa,writeit)
       mmax(2)=myid
       call MPI_ALLREDUCE(mmax,nmax,1,MPI_2REAL,MPI_MAXLOC,   &
                          MPI_COMM_WORLD,ierr)
+      !!$acc update device(mmax)
```

Comment:
Looks like this directive is not used. Maybe remove it?

Reply:
Thanks for the explanation. After looking through these loops, I guess the issue may come from the index for the maximum value, as we discussed before that the GPU code may not return the last index for the maximum value as  the CPU code does. A similar issue exists in the maxmin.F as well. Shall we convert them to a sequential run in a separate PR or in this PR?